webpack/assets/javascripts/react_app/API.js

```diff
@@ -30,9 +30,6 @@ export default {
       type: 'put',
       dataType: 'json',
       data: data,
-      success: function (response, textstatus, jqXHR) {
```

Comment:
this is exactly the purpose of this PR, to make immediate update to the state (optimistic concurrency).